src/Root_TEST.cc

```diff
@@ -65,6 +66,24 @@ TEST(DOMRoot, MoveAssignmentOperator)
   EXPECT_EQ("test_root", root2.Version());
 }
 
+TEST(DOMRoot, WorldName)
```

Comment:
Remove this change.

Reply:
https://github.com/gazebosim/sdformat/pull/997/commits/48698b0d1b7e87b165fdb2bc6e0dbca04d4cba75